src/sklearn_raster/features.py

```diff
@@ -294,7 +294,7 @@ def _get_attrs(
             set_attrs.update(new_attrs)
 
         if keep_attrs:
-            return set_attrs | attrs
+            return attrs | set_attrs
```

Comment:
Sorry, I totally missed this one when reviewing as well.  This makes sense, though.

Reply:
No worries! I just learned about this feature, but apparently didn't learn to use it correctly 😅